projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -3237,8 +3237,186 @@ public string FatherEthnicityLiteral
                     }
                     InputRaceAndEthnicityObsFather.Component.Add(component);
                 }
-
             }
         }
+
+        /// <summary>Mother's Education Level.</summary>
+        /// <value>the mother's education level. A Dictionary representing a code, containing the following key/value pairs:
+        /// <para>"code" - the code</para>
+        /// <para>"system" - the code system this code belongs to</para>
+        /// <para>"display" - a human readable meaning of the code</para>
+        /// </value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>Dictionary&lt;string, string&gt; elevel = new Dictionary&lt;string, string&gt;();</para>
+        /// <para>elevel.Add("code", "BA");</para>
+        /// <para>elevel.Add("system", VR.CodeSystems.EducationLevel);</para>
+        /// <para>elevel.Add("display", "Bachelor’s Degree");</para>
+        /// <para>ExampleBirthRecord.MotherEducationLevel = elevel;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Mother's Education Level: {ExampleBirthRecord.MotherEducationLevel["display"]}");</para>
+        /// </example>
+        [Property("Mother's Education Level", Property.Types.Dictionary, "Education Profiles", "Mother's Education Level.", true, VR.IGURL.EducationLevel, false, 32)]
+        [PropertyParam("code", "The code used to describe this concept.")]
+        [PropertyParam("system", "The relevant code system.")]
+        [PropertyParam("display", "The human readable version of this code.")]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='57712-2').value.coding", "")]
+        [FHIRSubject(FHIRSubject.Subject.Newborn)]
+        public Dictionary<string, string> MotherEducationLevel
+        {
+            get => GetObservationValue("57712-2");
+            set => SetObservationValue(value, "57712-2", CodeSystems.LOINC, "Highest level of education Mother", VR.ProfileURL.EducationLevel, NEWBORN_INFORMATION_SECTION);
```

Comment:
Do you also need to set the focus to the mother to differentiate this from the father education level observation?